src/Core/Prestashop.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core;
+
+class Prestashop
```

Comment:
Can you please add a quick description of this class please? Even if it's just about describing that it's just meant to be a "const container"

By the way you could add `private __construct()` method to emphasize (and even block) the fact that this lass should not be instanciated

Reply:
@jolelievre so, It can be abstract, right ?

Reply:
Related issue created https://github.com/PrestaShopCorp/TranslationTool/issues/120

Reply:
![image](https://user-images.githubusercontent.com/16455155/193252168-67c13780-a884-47a1-a5a0-4dcbb461de1f.png)